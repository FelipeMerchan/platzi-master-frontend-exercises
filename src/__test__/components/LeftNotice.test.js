import React from 'react'
import { shallow } from 'enzyme'
import { create } from 'react-test-renderer'
import '../../setupTest'
import LeftNotice from '../../components/LeftNotice'

describe('Probar el componente <LeftNotice />', () => {
  const leftNotice = shallow(<LeftNotice />)

  test('Render del componente LeftNotice',() => {
    expect(leftNotice.length).toEqual(1)
  })
  test('Render del tÃ­tulo principal', () => {
    expect(leftNotice.find('h1').text()).toEqual('Lorem ipsum dolor sit amet')
  })
  test('Render con el className text-center', () => {
    expect(leftNotice.find('h4').hasClass('text-center')).toBe(true)
  })
})

describe('LeftNotice Snapshot', () => {
  test('Comprobar la interfaz de usuario del componente LeftNotice', () => {
    const leftNotice = create(<LeftNotice />)
    expect(leftNotice.toJSON()).toMatchSnapshot()
  })
})