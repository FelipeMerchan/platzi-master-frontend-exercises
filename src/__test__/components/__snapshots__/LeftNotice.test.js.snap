// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeftNotice Snapshot Comprobar la interfaz de usuario del componente LeftNotice 1`] = `
<section
  className="LeftNotice"
>
  <h1
    className="text-center text-uppercase line-bottom"
  >
    Lorem ipsum dolor sit amet
  </h1>
  <h3
    className="text-center text-uppercase line-bottom"
  >
    Lorem ipsum
  </h3>
  <h4
    className="text-center line-bottom"
  >
    Lorem ipsum dolor sit amet, consectetuer adipiscing elit
  </h4>
  <h5
    className="text-center text-uppercase"
  >
    Lorem ipsum dolor
  </h5>
  <p
    className="text-justify"
  >
    Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat. Duis autem vel eum iriure dolor in hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi.
  </p>
  <p
    className="text-justify"
  >
    Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat. Duis autem vel eum iriure dolor in hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi.
  </p>
  <p
    className="text-justify"
  >
    Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat. Duis autem vel eum iriure dolor in hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi. praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi. praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi.  autem vel eum iriure dolor in hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero  autem vel eum iriure dolor in hendrerit in vulputate velit esse molesat vero.
  </p>
</section>
`;
